### PR TITLE
Fix glitch in website lists

### PR DESCRIPTION
**Pull request by iscreamcoke**
It fixes an inconsistency I had introduced in one of the website lists (forgot to pass on changes in background.js). As pointed out by @magnolia1234

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 // Cookies from this list are blocked by default
 var defaultSites = {
-  'Algemeen Dagblad': 'ad.nl',
+  'Algemeen Dagblad': 'ad.nl',
```
Comment (cimke) [inline background.js]: Looks like that you've left a space here

File: background.js [hunk 2]
```diff
@@ -401,18 +401,18 @@ chrome.webRequest.onCompleted.addListener(function(details) {
...
+      var cookie_domain = cookies[i].domain;
+      // hold specific cookie(s) from remove_cookies domains
+      if ((cookie_domain in remove_cookies_select_hold) && remove_cookies_select_hold[cookie_domain].includes(cookies[i].name)){
+        continue; // don't remove specific cookie
+      }
+      // drop only specific cookie(s) from remove_cookies domains
+      if ((cookie_domain in remove_cookies_select_drop) && !(remove_cookies_select_drop[cookie_domain].includes(cookies[i].name))){
+        continue; // only remove specific cookie
+      }
+      chrome.cookies.remove({url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path, name: cookies[i].name});
```
Comment (cimke) [inline background.js]: Just an object styling preference, however, I would suggest using [airbnb](https://github.com/airbnb/javascript#objects) to make code more readable. 
```suggestion
      chrome.cookies.remove({
          url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path,
          name: cookies[i].name
      });
```
or

```suggestion
      chrome.cookies.remove({ url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path, name: cookies[i].name });
```

File: background.js [hunk 3]
```diff
@@ -401,18 +401,18 @@ chrome.webRequest.onCompleted.addListener(function(details) {
...
-				continue; // don't remove specific cookie
-			}
-			// drop only specific cookie(s) from remove_cookies domains
-			if ((cookie_domain in remove_cookies_select_drop) && !(remove_cookies_select_drop[cookie_domain].includes(cookies[i].name))){
-				continue; // only remove specific cookie
-			}
-			chrome.cookies.remove({url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path, name: cookies[i].name});
-		}
+    for (var i=0; i<cookies.length; i++) {
+      var cookie_domain = cookies[i].domain;
```
Comment (cimke) [inline background.js]: As long as this one isn't going to be changed you should use `const`.

File: background.js [hunk 4]
```diff
@@ -401,18 +401,18 @@ chrome.webRequest.onCompleted.addListener(function(details) {
...
-			if ((cookie_domain in remove_cookies_select_hold) && remove_cookies_select_hold[cookie_domain].includes(cookies[i].name)){
-				continue; // don't remove specific cookie
-			}
-			// drop only specific cookie(s) from remove_cookies domains
-			if ((cookie_domain in remove_cookies_select_drop) && !(remove_cookies_select_drop[cookie_domain].includes(cookies[i].name))){
-				continue; // only remove specific cookie
-			}
-			chrome.cookies.remove({url: (cookies[i].secure ? "https://" : "http://") + cookies[i].domain + cookies[i].path, name: cookies[i].name});
-		}
+    for (var i=0; i<cookies.length; i++) {
```
Comment (cimke) [inline background.js]:
```suggestion
    for (let i=0; i<cookies.length; i++) {
```

**Comment by magnolia1234** [comment]
Irrelevant styling issues.

**Comment by cimke** [comment]
@magnolia1234 As long as it's an issue it's relevant to fix it.

**Comment by magnolia1234** [comment]
I would focus on functional improvements instead of 'styling preference'.

**Comment by iamadamdev** [comment]
Too much is going on in this PR.

If you want to fix a glitch in the sites do that only. It will make PRs easier to review.

If you want to fix styling changes that is fine, but please do it in a separate PR.

**Comment by magnolia1234** [comment]
Die Zeit and The Irish Times were not default options anymore after a changed label in background.js (and alphabetical reordering).
PS fixing merge conflicts in GitHub's online editor can also cause restyling though.

**Comment by iscreamcoke** [comment]
I just want to fix the glitch, the styling has been merged in the PR

**Comment by iscreamcoke** [comment]
fixed merging conflicts after PR #403

**Pull request merged**